### PR TITLE
Fixed goodbye message waiting for code review

### PR DESCRIPTION
**Pull request by ajgoranson**
"This is a temporary fix to adding a message for sign out it is not perfect and I want to add a message the same way we did in the labs with the confirmation delete message but I am having a few issues we can go over on Monday"

### CODE REVIEW

**Comment by claraj** [comment]
You don't need to make a PR if the work is not finished, push the branch and let your team/me know you'd like help. PRs are for work that is finished and needs to be double-checked before merge.  

For this issue, you'll need to figure out what currently happens when a user logs out. Take a look at the very end of settings.py. 

You'll probably need a new template & view to display a new page with a goodbye message. Make sure you only see the message immediately after a user has _just_ logged out and not when a user is generally just not logged in.

**Comment by ajgoranson** [comment]
Finished the Goodbye message with a reroute to a new URL, created a new view, URL, and template

File: lmn/templates/registration/goodbye.html [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{% extends 'lmn/base.html' %}
+
+{% block content %}
+
+
+
+<h2>Successfully singed out!!<h2>
```
Comment (claraj) [inline lmn/templates/registration/goodbye.html:7]: watch your spelling

File: lmn/views/views_users.py [hunk 2]
```diff
@@ -31,12 +31,25 @@ def register(request):
             if user:
                 login(request, user)
                 return redirect('user_profile', user_pk=request.user.pk)
+            elif user:
```
Comment (claraj) [inline lmn/views/views_users.py:34]: do you need this additional code?

**Comment by claraj** [comment]
I haven't run your code, ask at least one person on your team to run and test this. 

Also you need a unit test to verify the message does display immediately after a user logs out.

**Comment by johnnyvuanly** [comment]
Just reviewed the code and you already showed that it worked in Zoom. Merging

**Comment by claraj** [comment]
@johnnyvuanly this needs unit tests and there's other issues noted in my review that should have been fixed before merged

**Comment by claraj** [comment]
The empty js file should be removed too

**Pull request merged**